eslint.config.js

```diff
@@ -184,7 +184,11 @@ module.exports = [
       'yml/vue-custom-block/no-parsing-error': 'off'
     }
   },
-  eslintPlugin.configs['flat/recommended'],
+  {
+    name: 'ESLint Plugin',
+    ...eslintPlugin.configs['flat/recommended'],
+    files: ['**/*.ts']
+  },
```

Comment:
This change was necessary because otherwise this plugin would report errors for JSON files, which is never valid.